README.md

```diff
@@ -74,16 +77,17 @@ backport version of `libseccomp-dev` is required. See [travis.yml](.travis.yml)
 # `cni`, please follow instructions in their documents.
 make install.deps
 ```
-### Build and Install cri
-To build and install a version of containerd with the `cri` plugin enter the
+### Build and Install `cri`
+To build and install a version of containerd with the `cri` plugin, enter the
 following commands from your `cri` project directory:
 ```bash
 make
 sudo make install
 ```
+*NOTE: This is only for developer to build and test latest cri plugin.*
```

Comment:
The version of containerd built and installed from the containerd/cri/Makefile is only for testing purposes. The version tag carries the suffix "-test"

Reply:
Will do.

Reply:
Done